packages/pages/src/generate/features/features.ts

```diff
@@ -0,0 +1,66 @@
+import path from "path";
+import { CommandModule } from "yargs";
+import { Path } from "../../common/src/project/path.js";
+import { ProjectFilepaths } from "../../common/src/project/structure.js";
+import { getTemplateFilepaths } from "../../common/src/template/internal/getTemplateFilepaths.js";
+import { loadTemplateModules } from "../../common/src/template/internal/loader.js";
+import { createFeaturesJson } from "./createfeaturesjson.js";
+
+type FeaturesArgs = Pick<
+  ProjectFilepaths,
+  "templatesRoot" | "sitesConfigRoot" | "scope"
+>;
+
+const handler = async ({
+  templatesRoot,
+  sitesConfigRoot,
+  scope,
+}: FeaturesArgs): Promise<void> => {
+  const templateRootAbsolutePath = path.join(process.cwd(), templatesRoot);
+  const templateFilepaths = getTemplateFilepaths(
+    scope
+      ? [
+          new Path(path.join(templateRootAbsolutePath, scope)),
+          new Path(templateRootAbsolutePath),
+        ]
+      : [new Path(templateRootAbsolutePath)]
+  );
+  const templateModules = await loadTemplateModules(
+    templateFilepaths,
+    true,
+    false
+  );
+  const featuresFilepath = path.join(
+    process.cwd(),
+    sitesConfigRoot,
+    scope ?? "",
+    "features.json"
+  );
+  await createFeaturesJson(templateModules, featuresFilepath);
+};
+
+export const featureCommandModule: CommandModule<unknown, FeaturesArgs> = {
+  command: "features",
+  describe: "Generates features.json file",
+  builder: (yargs) => {
+    return yargs
+      .option("templatesRoot", {
+        describe: "The folder path where all template files live",
+        type: "string",
+        default: "src/templates",
+        demandOption: false,
```

Comment:
is there a config file where somebody can specify their settings for this stuff?

Reply:
using pages command line, there's currently no way for user to configure these options beside the new arguments we are adding. From vite-plugin, they can technically pass in a `ProjectStructureConfig` object containing whatever settings they want (e.g `yextSSG({ //user-config })`). But as Duval[ mentioned above](https://github.com/yext/pages/pull/248#discussion_r1004799768), we may want to restrict what user can pass in.